monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/RunManually/LocalManualRunOptions.js

```diff
@@ -45,12 +51,18 @@ const getContents = (props) => {
     }
   }
 
+  function getOtp() {
+    IslandHttpClient.get(APIEndpoint.agent_otp).then(res =>{
```

Comment:
Does this need to be called via `authFetch`? Or does `IslandHttpClient` handle that?

Reply:
`IslandHttpClient` uses `authFetch`.